src/mod_pushoff_apns.erl

```diff
@@ -62,10 +62,11 @@
          pending_timestamp :: erlang:timestamp(),
          retry_timestamp :: erlang:timestamp(),
          message_id :: pos_integer(),
-         gateway :: string()}).
+         gateway :: string(),
+         api_key :: string()}).
 
-init([CertFile, Gateway]) ->
-    ?INFO_MSG("+++++++++ mod_pushoff_apns:init, certfile = ~p, gateway ~p", [CertFile, Gateway]),
+init([CertFile, Gateway, ApiKey]) ->
+    ?INFO_MSG("+++++++++ mod_pushoff_apns:init, certfile = <~p>, gateway <~p>, ApiKey <~p>", [CertFile, Gateway, ApiKey]),
```

Comment:
it bugs me that we're mixing global configuration with backend specific configuration — note for later